app/controllers/evaluation_controller.rb

```diff
@@ -107,9 +112,12 @@ def upload_gpr
 
       flash[:notice] = "#{num_new_records} new GPRs imported. #{num_updated_records} evaluation GPRs updated."
       redirect_to evaluation_index_path
-    else
+    elsif params[:data_file] != nil
```

Comment:
Not sure what you mean; we added handling the case of uploading before selecting a file in both pica upload and GPR upload, which caused line numbers to shift